src/components/layouts/Layout.js

```diff
@@ -40,38 +40,54 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
   const [context, setContext] = useState(initialContext || 'homepage');
   const [menuVisible, setMenuVisible] = useState(false);
   const lastPartofPath = /[^/]*$/;
-  const currentPagePath = location ? location.pathname.match(lastPartofPath) : 'Home';
+  const currentPagePath = location
+    ? location.pathname.match(lastPartofPath)
+    : 'Home';
   const currentPageName = currentPagePath
     .toString()
     .replace(/-/g, ' ')
     .split(' ')
-    .map(word => word.charAt(0).toUpperCase() + word.substring(1))
+    .map((word) => word.charAt(0).toUpperCase() + word.substring(1))
     .join(' ');
 
   return (
     <StaticQuery
       query={graphql`
-      {
-        allSparkIconSet {
-          edges {
-            node {
-              internal {
-                content
+        {
+          allSparkIconSet {
+            edges {
+              node {
+                internal {
+                  content
+                }
               }
             }
           }
         }
-      }
-    `}
-      render={data => (
+      `}
+      render={(data) => (
         <>
-          <div className="sprk-o-Box docs-c-SkipNav">
+          <div className="sprk-o-Box docs-c-SkipNav sprk-u-BackgroundColor--white">
             <SprkLink
-              href="#main_content" additionalClasses="docs-c-SkipNav__link"
+              href="#main_content"
+              additionalClasses="docs-c-SkipNav__link"
             >
               Skip to Main Content
             </SprkLink>
           </div>
+          <div
+            className="sprk-u-BackgroundColor--black sprk-u-Color--white
+            sprk-u-pvm
+            sprk-u-AbsoluteCenter"
+          >
+            <a
+              href="https://www.sparkdesignsystem.com/"
+              className="docs-c-Banner--link sprk-u-mlm"
+            >
+              For designs launching AFTER July 14, 2021, please reference the
```

Comment:
This whole sentence shouldn't be a link.

Reply:
Updated the message, and added the link to "latest release of Spark" only.